tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -23,6 +23,9 @@ void *exp_user_data = (void *)199;
 struct counter_alarm_cfg cntr_alarm_cfg;
 struct counter_alarm_cfg cntr_alarm_cfg2;
 
+#define DELAY_US 1200
+#define RANGE 500
```

Comment:
where is number from? the counter range in us?

Reply:
Yes, the range is in us... I'll rename the macro after the checks pass.

Reply:
the problem is not only the naming, it is the drift range is way of too large, this means the counter can be 50% inaccurate.

Reply:
We do not care about precision. Our goal here is to create a testcase which tests if the counter starts and does something with drivers that do not support the alarm feature. Some of our driver do not let us even set the top value, so the test_set_top_value_without_alarm_instance would be skipped and no test would validate the correct behavior.